src/schemas/ba-sak/begrunnelse/nasjonaleTriggere/utils.ts

```diff
@@ -1,4 +1,17 @@
-import { hentNasjonaltFeltRegler, erNasjonalBegrunnelse } from '../utils';
+import { Behandlingstema, NasjonalBegrunnelse, Begrunnelse } from '../typer';
+import { BegrunnelseDokumentNavn } from '../../../../util/typer';
+
+export const erNasjonalBegrunnelse = (document: Begrunnelse): document is NasjonalBegrunnelse =>
```

Comment:
Legger disse i denne utilsfilen for å ha likt pattern på tvers av de tre behandlingstemaene